### PR TITLE
Test/form submitting

### PR DESCRIPTION
**Pull request by szymonkadas**
# Summary: 
Implemented tests for form submission. 

# Notes:
for some reason i've had problems with userEvent, it simply didn't trigger anything -.-

### CODE REVIEW

File: src/pages/PaniniCreator.test.tsx [hunk 1]
```diff
@@ -0,0 +1,448 @@
...
+      });
+
+      test(`value of ${PaniniNames.bread} is indeed default on start`, () => {
+        expect(inputElement.value).toBe(defaultVal);
+      });
+
+      test(`value of ${PaniniNames.bread} is changed after user interaction`, async () => {
+        const defaultValIndex = breadVariants.findIndex((val) => val === defaultVal);
+        // swipe left and back or swipe right and back.
+        if (defaultValIndex > 0) {
```
Comment (Sumick) [inline src/pages/PaniniCreator.test.tsx]: **nitpick:** This all logic not needed - you could simplify it just to:
1. expect that default value is set (as you do before)
2. click right or left arrow
3. check if value is not the same as default one

**Review by Sumick — CHANGES_REQUESTED** [review]
Im having really huge problems with reading these tests - they are written in very imperative way, test file is huge, and you are missing code splitting into some smaller parts, like helpers for finding elements. For example:

Something like this tells me nothing, but if it was wrapped into `findDressingElements`, what would make it much more clear, and you could reuse it in other tests.

```ts
leftSwipeButtons = Array.from(lodashGet(formDefaultValues, PaniniNames.dressing), (val, index) => {
          return screen.getByTestId(`${PaniniNames.dressing}${index}-swipeLeftButton`);
        });
```

Another thing is that you need to test only these things that are required to check if scenario works - you are going into too much details which should be covered by unit tests, not integration test. You don't need to test each possible way of interacting with element. 

To sum up, your current test file is a mix of unit, and integration testing. You don't need to cover scenarios like "swipe fields values are changed after user interaction". I've posted exact test scenarios in task requirements, so you can stick to them, and it will be totally enough. For example, if you have to test "It should be able to change default values of each form element before clicking Place order button", you can just do:

1. Render parent component that makes possible for your form to work
2. Find all inputs, and form controls that change values
3. Check if these values are the same as default, initial form value
4. Then interact with each of elements by clicking on them for example
5. Then expect that sent payload is different than default, initial form values

Main objective for this PR is to:

- [x] Simplify these tests
- [x] Make them more readable, and maintainable by splitting some logic into smaller helper functions if needed
- [x] Remove all unit testing logic eg. expects that checks if something was clicked or not clicked
- [x] Focus on these 3 scenarios listed in task description. You basically should have 3 tests, nothing more is needed

**Comment by szymonkadas** [comment]
One note: 
Didn't knew yet how to mock payload with these changed values, so it's still checking if the fields are changed. If that's not okay, then i'll get back to this issue when i''ll figure out how to mock payload (with changes aswell).

**Review by Sumick — APPROVED** [review]
Looks good!

**Pull request merged**